src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1475,10 +1475,25 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 			ins->inst_c1 = arg0_type;
 			return ins;
 		} else {
-			if (!COMPILE_LLVM (cfg))
-				// FIXME:
-				return NULL;
-			return emit_simd_ins_for_sig (cfg, klass, OP_VECTOR_IABS, -1, arg0_type, fsig, args);
+			if (COMPILE_LLVM (cfg))
+				return emit_simd_ins_for_sig (cfg, klass, OP_VECTOR_IABS, -1, arg0_type, fsig, args);
+
+			// SSSE3 does not support i64
+			if (arg0_type == MONO_TYPE_I8) {
+				MonoInst *zero = emit_xzero (cfg, klass);
+				MonoInst *neg = emit_simd_ins (cfg, klass, OP_XBINOP, zero->dreg, args [0]->dreg);
+				neg->inst_c0 = OP_ISUB;
+				neg->inst_c1 = MONO_TYPE_I8;
+				MonoInst *ins = emit_simd_ins (cfg, klass, OP_XBINOP, args [0]->dreg, neg->dreg);
+				ins->inst_c0 = OP_IMAX;
+				ins->inst_c1 = MONO_TYPE_I8;
```

Comment:
Is this being handled correctly on the backend?

There isn't a vector max for `i64` until AVX512F (at which point there is also `pabsq` available) so this `OP_IMAX` has to have some fallback to emulate with a `CompareGreaterThan` and then a `ConditionalSelect`.

`pcmpgtq` is itself only available on SSE4.2 or later, so it also has to be emulated as well on downlevel hardware, doing something like: https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/gentree.cpp#L21409-L21459

Reply:
https://github.com/dotnet/runtime/blob/af874399a0de79c9364f3f0e8cebbd0c60e16dfa/src/mono/mono/mini/mini-amd64.c#L3738-L3767

I think this is handled here